client/v2/autocli/msg.go

```
@@ -228,3 +236,74 @@
 
 	return clienttx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), proposal)
 }
+
+// generateOrBroadcastTxWithV2 generates or broadcasts a transaction with the provided messages using v2 transaction handling.
+//
+//nolint:unused // It'll be used once BuildMsgMethodCommand is updated to use factory v2.
```

golangci-lint (1.62.2)

directive `//nolint:unused // It'll be used once BuildMsgMethodCommand is updated to use factory v2.` is unused for linter "unused"

(nolintlint)